resources/api.json

```diff
@@ -0,0 +1,3 @@
+{
+  "client_id":"1018177563991-url06g1vfid7kveu6eb26rqkka73milf.apps.googleusercontent.com"
+}
```

Comment:
Nah, it's specifically a key used for js calls to a Google API, it'd be visible client-side regardless.